contracts/src/interfaces/IEigenDAStructs.sol

```diff
@@ -82,8 +82,8 @@ struct BlobHeaderV2 {
     uint16 version;
     bytes quorumNumbers;
     BlobCommitment commitment;
-    uint32 salt;
     bytes32 paymentHeaderHash;
+    uint32 salt;
 }
 
 struct BlobCommitment {
```

Comment:
Last name discrepancy (I think) between onchain<>offchain is `BlobCommitment.dataLength`. In the offchain code its just called `BlobCommitment.length`. Can we make them the same? Either `length` (to match `lengthProof` and `lengthCommitment`) or `blobLength`, but not `dataLength` because we've already moved away from the "data" naming to more precise naming.